ww3tools/ww3tools.yaml

```diff
@@ -0,0 +1,59 @@
+# Configuration file for WW3-tools
+# https://github.com/NOAA-EMC/WW3-tools
+
+# text string for identification. This will be included in the name of the output files.
+ftag: ww3tools
+# output path, where results will be saved
+path_out: /work/noaa/marine/ricardo.campos/test
+
+# Data paths where the observations are saved
+path_alt: /work/noaa/marine/ricardo.campos/data/AODN/altimeter
+path_ndbc: /work/noaa/marine/ricardo.campos/data/buoys/NDBC/ncformat/wparam
+path_copernicus: /work/noaa/marine/ricardo.campos/data/buoys/Copernicus/wtimeseries
+
+# WAVEWATCHIII field output, for the scripts to obtain the latitude and longitude
+# grid_info: /work/noaa/marine/ricardo.campos/work/ww3tools/tests/ww3gfs_met5_2022061600.nc
+grid_info: /work/noaa/marine/ricardo.campos/work/ww3tools/tests/ops.glwu.20220601.t19z.nc
+# Grid format: Unstructured (0) or Structured (1)
+grid_format: 1
+
+# General quality control parameters. Basic range test
+#  Significant wave height (m) min and max:
+hsmin: 0.
+hsmax: 20.
+#  Wind speed (m/s) min and max:
+wspmin: 0.
+wspmax: 80.
+
+# netcdf format when saving outputs
+fnetcdf: 'NETCDF4'
+
+# Parameters for ProcSat_Altimeter.py
+# target space for collocation. Model (1), Altimeter (2), I don't know (0)
```

Comment:
Is 0 a valid option? What happens for zero?

Reply:
Not yet. In the future, I'd like to create a decision based on the resolution of the model/observations